lua/base16-colorscheme.lua

```diff
@@ -370,6 +371,35 @@ function M.setup(colors, config)
     hi.NotifyDEBUGBody = 'Normal'
     hi.NotifyTRACEBody = 'Normal'
 
+    hi.CmpItemAbbrDeprecated  = { guifg = M.colors.base03, guibg = nil, gui = "strikethrough", guisp = nil }
+    hi.CmpItemAbbrMatch       = { guifg = M.colors.base0D, guibg = nil, gui = "none", guisp = nil }
+    hi.CmpItemAbbrMatchFuzzy  = { guifg = M.colors.base0D, guibg = nil, gui = "none", guisp = nil }
+    hi.CmpItemKindFunction    = "TSFunction"
+    hi.CmpItemKindMethod      = "TSFunction"
+    hi.CmpItemKindConstructor = "TSFunction"
+    hi.CmpItemKindClass       = "TSType"
+    hi.CmpItemKindEnum        = "TSType"
+    hi.CmpItemKindEvent       = "TSType"
+    hi.CmpItemKindInterface   = "TSType"
+    hi.CmpItemKindStruct      = "TSType"
+    hi.CmpItemKindVariable    = "TSVariable"
+    hi.CmpItemKindField       = "TSVariable"
+    hi.CmpItemKindProperty    = "TSProperty"
+    hi.CmpItemKindEnumMember  = "TSType"
+    hi.CmpItemKindConstant    = "TSConstant"
+    hi.CmpItemKindKeyword     = "TSKeyword"
+    hi.CmpItemKindModule      = "TSType"
+    -- hi.CmpItemKindValue
+    -- hi.CmpItemKindUnit
+    -- hi.CmpItemKindText
+    -- hi.CmpItemKindSnippet
+    -- hi.CmpItemKindFile
+    -- hi.CmpItemKindFolder
+    -- hi.CmpItemKindColor
+    -- hi.CmpItemKindReference
+    -- hi.CmpItemKindOperator
+    -- hi.CmpItemKindTypeParameter
```

Comment:
Remove this

Reply:
Okay.

Reply:
Actually, I think it's better to link them to "normal" or something similar on order to override whatever the previous colors theme have set.